specification/common-types/resource-management/v5/networksecurityperimeter.json

```
@@ -0,0 +1,337 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-09-15-preview",
+    "title": "Common types for network security perimeters based on a common API specification"
+  },
+  "definitions": {
+    "PublicNetworkAccess": {
+      "type": "string",
+      "description": "Allow, disallow, or let Network Security Perimeter configuration to evaluate public network access to Storage Account. Value is optional but if passed in, must be 'Enabled', 'Disabled' or 'SecuredByPerimeter'.",
+      "enum": [
+        "Enabled",
+        "Disabled",
+        "SecuredByPerimeter"
+      ],
+      "x-ms-enum": {
+        "name": "PublicNetworkAccess",
+        "modelAsString": true
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationListResult": {
+      "description": "Result of a list NSP (network security perimeter) configurations request.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of network security perimeter results.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkSecurityPerimeterConfiguration"
+          },
+          "nextLink": {
+            "description": "The link used to get the next page of results.",
+            "type": "string",
+            "format": "uri",
+            "readOnly": true
+          }
+        }
+      },
+      "NetworkSecurityPerimeterConfiguration": {
+        "description": "Network security perimeter (NSP) configuration resource",
+        "type": "object",
+        "allOf": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
```

Relative links should be relative to the current file.
```suggestion
            "$ref": "types.json#/definitions/ProxyResource"
```